plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/styles/Fonts.js

```diff
@@ -90,48 +90,48 @@ Ext.define('NX.view.dev.styles.Fonts', {
       {
         xtype: 'panel',
         title: 'Faces',
-        ui: 'subsection',
+        ui: 'nx-subsection',
         layout: {
           type: 'hbox',
           defaultMargins: {top: 0, right: 20, bottom: 0, left: 0}
         },
         items: [
-          faceExample('Proxima Nova Regular', 'proxima-nova-regular'),
-          faceExample('Proxima Nova Bold', 'proxima-nova-bold'),
-          faceExample('Courier New', 'courier-new-regular')
+          faceExample('Proxima Nova Regular', 'nx-proxima-nova-regular'),
```

Comment:
not sure if we want to do this, these are not really 'our' fonts but don't feel strongly either way